### DISCUSSION TITLE
Request for auto popup/session renewal once session expires

### DISCUSSION THREAD

**Discussion by lakshaykaura** [sushrut111/cowin-automation-extn#45] [General]
Hi Team,
I see we are maintaining session timer on tab, and it goes in negative values as well. Request you to auto click on logout and login again to get OTP so session can be extended and user should not be asked to login with OTP when slot was found as it waste time. Better we should never go in negative time.

**Comment by sushrut111** [comment]
I will give it a thought, thanks!
